pkg/http.go

```diff
@@ -23,6 +23,12 @@ type Response struct {
 	Type   string `json:"type,omitempty"`
 }
 
+var version string // This will be injected at build time
+
+type VersionResponse struct {
+	Version string `json:"version"`
+}
```

Comment:
add the commit sha

Reply:
type VersionResponse struct {
    Version   string `json:"version"`
    CommitSHA string `json:"commit_sha"`
}